WordPress/Classes/ViewRelated/System/Notices/NoticeStyle.swift

```diff
@@ -68,6 +68,28 @@ public struct NormalNoticeStyle: NoticeStyle {
     public let dismissGesture: NoticeDismissGesture? = nil
 }
 
+public struct InAppUpdateNoticeStyle: NoticeStyle {
+    public let attributedMessage: NSAttributedString?
+
+    init(attributedMessage: NSAttributedString? = nil) {
+        self.attributedMessage = attributedMessage
+    }
+
+    // Return new UIFont instance everytime in order to be responsive to accessibility font size changes
+    public var titleLabelFont: UIFont { return WPStyleGuide.fontForTextStyle(.subheadline, fontWeight: .semibold) }
+    public var actionButtonFont: UIFont? { return WPStyleGuide.fontForTextStyle(.headline) }
+    public var cancelButtonFont: UIFont? { return WPStyleGuide.fontForTextStyle(.body) }
+
+    public let layoutMargins = UIEdgeInsets(top: 13.0, left: 16.0, bottom: 13.0, right: 16.0)
+
+    public var isDismissable = false
```

Comment:
Does `isDismissable = false` mean that you have to tap "Update"? Does it match the requirements?

Reply:
I added a cancel button. The notice won't be dismissed until the user taps either the cancel or the update button.

<img src="https://github.com/wordpress-mobile/WordPress-iOS/assets/6711616/7a3a42cf-4ecc-4f96-90ca-47159cb7720f" width=200>